example/src/components/ChatboxWrapper.tsx

```diff
@@ -1,23 +1,30 @@
-import { FC } from 'react';
+import { FC, useEffect, useState } from 'react';
 import { List } from 'immutable';
 import Chatbox, {
   ChatMessage,
   PartialChatMessage,
   ImmutableMember,
   Member,
+  AvatarHookType,
 } from '@graasp/chatbox';
 import { MUTATION_KEYS } from '@graasp/query-client';
 import { useMutation, hooks } from '../config/queryClient';
 import { PartialNewChatMessage } from '../../../src';
-import { AvatarHookType } from '../../../src/types';
+import { HEADER_SIZE } from '../config/constants';
 
-type Props = {};
-
-const ChatboxWrapper: FC<Props> = () => {
-  // this is the id of the item to which the chat is attached (folder, document ...)
-  const chatId = '39370f67-2153-4ab9-9679-b1966542d27d';
-  const lang = 'fr';
+type Props = {
+  chatId: string;
+  lang?: string;
+  showHeader?: boolean;
+  showAdminTools?: boolean;
+};
 
+const ChatboxWrapper: FC<Props> = ({
+  chatId,
+  lang = 'en',
```

Comment:
I see you have a default lang in your constants, but you're using english here, is there a reason?